docs/docs/signing.md

```diff
@@ -79,84 +99,63 @@ const req = {
 const sig = await client.sign(req)
 ```
 
-### 💾 Calldata Decoding
+## 💾 Calldata Decoding
 
-:::note
-All available calldata decoding options will be documented in this section. More may be added as time goes on.
-:::
+> NOTE: All available calldata decoding options will be documented in this section. More may be added as time goes on.
 
 Certain transaction decoder types may support calldata decoding for request data. You can use this feature by including "calldata decoder data" (explained shortly) in a general signing request using the `decoder` request param:
 
 ```ts
-req.decoder = <calldata decoder data>
+req.data = {
+  payload: <Raw message to be signed, e.g. serialized transaction>,
+  decoder: <Optional serialized information about decoding the payload>
+}
 await client.sign(req);
 ```
 
 If you include a valid calldata decoder, the appearance of the transaction's data on the user's Lattice should transform from a raw hex string to a markdown-style version which displays the function name, parameter names, and values.
 
-#### Storing Calldata Decoders
+### 1️⃣ EVM Calldata Decoding
```

Comment:
Do we want to include any images of decoded data vs. non-decoded? We have some in progress.

Reply:
Looks like [that ticket](https://github.com/GridPlus/gridplus-io/issues/18) isn't done yet. I created [an issue](https://github.com/GridPlus/gridplus-sdk/issues/468) to update images in the sdk docs in the future.